services/QuillCMS/db/views/graded_responses_v01.sql

```diff
@@ -0,0 +1,4 @@
+-- Note, if you add or remove columns for the tables, you'll need a migration
```

Comment:
Scenic has [very specific commands ](https://github.com/scenic-views/scenic)for updating existing views, especially long running ones. How about linking to them here, for future readers?

Reply:
Good point! I've added comments.